.github/pull_request_template.md

```diff
@@ -5,10 +5,15 @@ What GitHub Desktop issue does this PR address? (for example, #1234)
 Closes #[issue number]
 
 ## Description
-
+<!--
+What changes did you make to the code?
```

Comment:
Since pull requests can contain changes to more than just code (i.e. images, documentation, etc) I think it'd be a good idea to keep this comment more generic, how about something like this?

```suggestion
A summary of the changes made along with any other information that would be helpful to a reviewer such as potential tradeoffs or alternative approaches you considered.
```

Reply:
Thanks, committed